sdks/python/apache_beam/runners/common.py

```diff
@@ -229,7 +229,7 @@ def invoke_timer_callback(self, user_state_context, key, window, timestamp):
         kwargs[kw] = user_state_context.get_state(state_spec, key, window)
       for kw, timer_spec in self.timer_args_to_replace.items():
         kwargs[kw] = user_state_context.get_timer(
-            timer_spec, key, window, None, None)
+            timer_spec, key, window, timestamp, None)
```

Comment:
The paneinfo should also be passed in here. You can get paneinfo from `timer.paneinfo`